netdata.spec.in

```diff
@@ -154,18 +154,19 @@ Requires: netdata-plugin-apps
 Requires: netdata-plugin-pythond
 Requires: netdata-plugin-go
 Requires: netdata-plugin-debugfs
-%if 0%{?_have_freeipmi}
-Requires:   netdata-plugin-freeipmi
-%endif
+Requires: netdata-plugin-chartsd
+Requires: netdata-plugin-slabinfo
+Requires: netdata-plugin-perf
 %if 0%{?_have_nfacct}
 Requires:   netdata-plugin-nfacct
 %endif
-%if 0%{?centos_ver} != 6 && 0%{?centos_ver} != 7
-Requires:   netdata-plugin-cups
+%if 0%{?_have_freeipmi} && 0%{?centos_ver} != 6 && 0%{?centos_ver} != 7 && 0%{?amazon_linux} != 2
```

Comment:
nit-pick: I think we need to update the `_have_freeipmi` macro. It looks like this change belongs in there.

Reply:
`Suggests` is for optional deps that aren’t installed by default (which is what we want here). The problem is that the version of RPM on CentOS 7 (and AL 2) is too old to support it, so we can’t list it there. We still build the package (which is what `_have_freeipmi` controls), we just need to not reference it as a dependency at all.